plancton/__init__.py

```diff
@@ -263,7 +263,8 @@ def _create_container(self):
                            "Devices"     : [ dict(zip([ "PathOnHost", "PathInContainer",
                                                         "CgroupPermissions" ], x.split(":", 2)))
                                              for x in self.conf["devices"] ],
-                           "CapAdd"      : self.conf["capabilities"]
+                           "CapAdd"      : [ x.lstrip("+") for x in self.conf["capabilities"] if x and x[0]=="+" ]
```

Comment:
This condition is not backwards-compatible as it does not catch the case of missing `+`.

    [ x.lstrip("+") for x in self.conf["capabilities"] if x and x[0]!="-" ]